### ISSUE TITLE
Register-PSFTaskEngineTask -ResetTask Bug

### ISSUE THREAD

**Issue by Rick-2CA** [PowershellFrameworkCollective/psframework#372]
Attempting to utilize the `-ResetTask` parameter with `Register-PSFTaskEngineTask`.  Received the following error:

```
The property 'Status' cannot be found on this object. Verify that the property exists and can be set.
At C:\Program Files\WindowsPowerShell\Modules\PSFramework\1.0.19\PSFramework.psm1:7671 char:4
+             $task.Status = 'Pending'
```

I believe the `$task.Status` reference should be `$task.State`

**Comment by FriedrichWeinmann** [comment]
Bug found and fixed, will be resolved in the next release

**Comment by FriedrichWeinmann** [comment]
Resolved since 1.0.33
